From e0682babb3de08a9a605f2adc54231162df6262f Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Sun, 10 Jun 2018 20:06:24 +0100
Subject: [PATCH] flo: Inform Trust about legacy encryption support

 * Encryption causes noticeable performances loss, because HW crypto
   is not suported.

Change-Id: I4336c0cd050584020780d8ff97f454e40c7e6125
---
 overlay-lineage/lineage-sdk/lineage/res/res/values/config.xml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/overlay-lineage/lineage-sdk/lineage/res/res/values/config.xml b/overlay-lineage/lineage-sdk/lineage/res/res/values/config.xml
index 8dd6df6..f7bdaea 100644
--- a/overlay-lineage/lineage-sdk/lineage/res/res/values/config.xml
+++ b/overlay-lineage/lineage-sdk/lineage/res/res/values/config.xml
@@ -59,4 +59,8 @@
          For example, a device with Home, Back and Menu keys would set this
          config to 7. -->
     <integer name="config_deviceHardwareWakeKeys">64</integer>
+
+    <!-- Indicate whether encryption causes significant performances loss.
+         This MUST NOT be set to true on devices produced in 2016 or later -->
+    <bool name="config_trustLegacyEncryption">true</bool>
 </resources>
